project/plugins.sbt

```diff
@@ -5,12 +5,10 @@ logLevel := Level.Warn
 resolvers += "Typesafe repository" at "http://repo.typesafe.com/typesafe/releases/"
 
 // Use the Play sbt plugin for Play projects
-addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.3.8")
+addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.4.6")
```

Comment:
Also copy from swagger-play24 but use latest play sbt-plugin 2.4.6 instead of 2.4.0